src/components/Media/Media.scss

```diff
@@ -13,4 +13,12 @@ $block: '.#{$ns}Media';
         display: flex;
         align-items: center;
     }
+
+    &_border {
+        border: 1px solid var(--g-color-line-generic);
+    }
+
+    &_rounded {
```

Comment:
It was rounded before if I am not mistaken?

Reply:
PR was completely refactored. In discussion we came up with the solution that there should be an mixin added to media component
This decision made this comment non-relevant